engine/server/engine/enclave_manager/enclave_creator.go

```diff
@@ -75,6 +82,24 @@ func (creator *EnclaveCreator) CreateEnclave(
 		}
 	}()
 
+	// only create log collector for backend as
+	var shouldDeleteLogsCollector bool
```

Comment:
would it make sense to set `shouldDeleteLogsCollector` to `true` already here? Wondering if it opens up to bugs to only set it later?

Reply:
good idea